test.sh

```diff
@@ -0,0 +1,10 @@
+node ios/forceios.js create --apptype=native --appname=App1 --packagename=com.acme.app1 --organization=Acme1 --outputdir=app1 --templaterepourl
```

Comment:
This file will go soon (once test_force.js is fixed).
For now I used it for local manual testing.